english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/wishes.md

```diff
@@ -2,4 +2,5 @@ The little red fellow kind of creeps you out
  but you ask him to accompany you to the tower
   because these wishes might just come in handy ...
 
-[Wish that you and him teleport to the tower](./teleport/teleport.md)
\ No newline at end of file
+[Wish that you and him teleport to the tower](./teleport/teleport.md)
+[Wish that you and him get an eagle to ride on](./eagle/eagle.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.